### DISCUSSION TITLE
Automatic Configuration of Janssen server

### DISCUSSION THREAD

**Discussion by tawaren** [JanssenProject/jans#8326] [Q&A]
We currently use the Monolith for developing and testing our Janssen server.
Currentlly after starting the docker container from a blank state, I wait until the installation script finished and then manually use the   Janssen CLI (/opt/jans/jans-cli/config-cli.py) to upload the interception scripts and set some configuration.

However, we would like to automate this, but I'm a bit lost how to do this. Is their a way to just provide some configruations, that are automatically applied during install or right afterwards?

If not I will have to write some code that obtains the access token and use CURL to make the configuration.
This would run in another docker container once janssen started and install is done.
But I was just wondering if their is a more elegant solution to do this with the Monolith.

**Comment by nynymike** [comment]
Couldn't you write a bash or python script that calls the `config-cli.py` commands you need?  You could also call the config-api directly using `curl`.  But it would be nice to have an example how to do this in the docs, as there are some tricks, like storing the client creds or user creds. Let us know if you can figure it out, or else we'll add some more notes to the docs if it proves too challenging.

**Comment by ossdhaval** [comment]
Closing this as a stale discussion. Please comment if this is still a concern.
